m2e-configurator/net.revelc.code.formatter/src/net/revelc/code/formatter/connector/internal/FormatterProjectConfigurator.java

```diff
@@ -125,14 +126,20 @@ private InputStream readConfigFile(Formatter formatter, ProjectConfigurationRequ
             javaConfigFile = formatter.getDefaultPath();
         }
 
-        IFile cfgFile = request.getProject().getFile(javaConfigFile);
-
-        if (!cfgFile.exists()) {
-            throw new CoreException(
-                    new Status(IStatus.CANCEL, FormatterCore.PLUGIN_ID, "Configuration file not found!"));
+        File cfgFile = new File(javaConfigFile);
+        if (!cfgFile.isAbsolute()) {
```

Comment:
so, if the file exists we will look for another one that may or may not exists?!

:-1: